src/Tasks/DownloadFile.cs

```diff
@@ -60,6 +60,11 @@ public sealed class DownloadFile : TaskExtension, ICancelableTask
         [Required]
         public string SourceUrl { get; set; }
 
+        /// <summary>
+        /// Gets or sets the number of seconds to wait before the request times out.
+        /// </summary>
+        public double Timeout { get; set; } = 100;
```

Comment:
I'd prefer this to be named `TimeoutSec` to avoid any sort of confusion (debatable).

Reply:
Will change to milliseconds to be consistent with https://github.com/dotnet/msbuild/blob/18fc0b3dd9dfd7cbb83272f13de4a89e8aa30531/src/Utilities/ToolTask.cs#L210-L211

Reply:
Alright. Since there's already a precedent of `int Timeout` being set in ms, that's okay then. I guess.

Reply:
There other precedence for this parameter in Arcade. There it is named `TimeoutInSeconds`
https://github.com/dotnet/arcade/blob/4000024394df3049886c50e54ad0a2b903221ef0/src/Microsoft.DotNet.Arcade.Sdk/src/DownloadFile.cs#L38

Reply:
`Copy` also has `RetryDelayMilliseconds`, but there are a lot more `ToolTask`s than that, so I like just `Timeout` too.

Reply:
In retrospect I start liking `TimeoutInSeconds` better, because I cannot get case for milliseconds precision in build system, and having 3 zeros in build script would confuse. For example 10000 and 100000 milliseconds looks very similar, in comparison to 10 and 100 seconds.